src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -23,7 +23,7 @@
       "Size": 2676
     },
     "assemblies/UnnamedProject.dll": {
-      "Size": 3171
+      "Size": 3535
```

Comment:
Adding `Nullable=enable`, increased this assembly's size enough to fail the test...
```
Error: apkdiff: File 'assemblies/UnnamedProject.dll' has changed by 358 bytes (10.14 %). This exceeds the threshold of 5.00 %.
```